backend/routes/items.js

```diff
@@ -0,0 +1,73 @@
+const express = require("express");
+const { Item, Batch } = require("../models/item");
+const router = express.Router();
+const { toUTCMidnight } = require("../helpers/dateHelpers");
+
+router.get("/getAllItems", (req, res, next) => {
+  Item.find({}, (err, items) => {
+    if (err) {
+      console.error(err);
+    }
+    res.send(items, 200);
+  });
+});
+
+router.post("/addItem", (req, res, next) => {
+  // validate body params
+  const itemName = req.body["itemName"];
+  const itemId = req.body["itemId"];
+  const weight = req.body["weight"];
+  const today = toUTCMidnight(new Date());
```

Comment:
Is this getting today's date? Don't we want the expiration date here, which will be passed to you from the form? I guess depending on what design team answers regarding how the frontend calendar looks, we might need to keep track of what day the batch was added AND the expiration date. So maybe our batch needs the fields startDate and expirationDate. Can you update the model, and reflect that change in this file as well?